authenticator/views.py

```diff
@@ -1,19 +1,74 @@
 #Django
 from django.shortcuts import render
-from django.contrib.auth import authenticate, login
-from django.contrib.auth.backends import BaseBackend
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import User
 
+#Local model
+from authenticator.models import Personal_data
+
+
+#User creation in admin 
+def user_creation(request):
+
+    if request.method == 'POST':
+
+        username = request.POST['username']
+        password = request.POST['password']
+        password_conf = request.POST['password_conf']
+
+        if password != password_conf:
+            return f'Error'
+        else:
+            return f'Password accepted'
```

Comment:
Why return `f'Password accepted'` here? Wouldn't that finish the method's execution?